### ISSUE TITLE
md.drive(system) error! 

### ISSUE THREAD

**Issue by adnanphyqau** [ubermag/workshop#75]
Hello, I am using the the notebook "Spatially varying parameters 2" of workshop! When i run the driver it give me the error as given in the attached .png! 
P.S: the driver is working in other notebooks but except this on! Please guide me.
![image](https://user-images.githubusercontent.com/32726982/115455540-76ade000-a1f8-11eb-848a-ad83ba9579b9.png)

**Comment by marijanbeg** [comment]
Hi @adnanphyqau, thank you for your question. From the error message you got and the tutorial you are running, I can see two things:

1. You are running Ubermag on Windows.
2. The tutorial includes DMI term with crystallographic class T.

Crystallographic class T DMI is not available on Windows. However, this can be fixed by installing Docker. Please have a look at this tutorial: https://github.com/ubermag/workshop/blob/master/tutorials/choosing-runner.ipynb

In summary, you can install Docker (https://www.docker.com/) and that's it. The first time you run a simulation, it is going to take some time to download the Ubermag image from the cloud, but all subsequent simulations are as fast as running them without Docker.

I am going to close this issue, but please feel free to post comments if this does not solve your issue.

**Comment by adnanphyqau** [comment]
Hi Marijanbeg, thanks for your valuable comments/suggestion. 
I tried but the problem still remains. 
looks like docker is not working (attached), i installed docker desktop when i click on its icon give me the error of Hyper-V is not working (attached). I dnt know how fix this issue. Can you please give some suggestions? 

![image](https://user-images.githubusercontent.com/32726982/115479175-3d886680-a21e-11eb-9bc7-c8f5cf028881.png)
![image](https://user-images.githubusercontent.com/32726982/115479267-73c5e600-a21e-11eb-98db-9a8d62c1ddd1.png)
